testing/visual-regression/backstop_data/engine_scripts/puppet/onReady.js

```diff
@@ -1,7 +1,16 @@
-module.exports = async (page, scenario, vp) => {
-    console.log('SCENARIO > ' + scenario.label);
-    await require('./overrideCSS')(page, scenario);
-    await require('./clickAndHoverHelper')(page, scenario);
+module.exports = async (page, scenario) => {
+  console.log('SCENARIO > ' + scenario.label);
+  await require('./overrideCSS')(page, scenario);
+  await require('./clickAndHoverHelper')(page, scenario);
 
-    // add more ready handlers here...
+  /**
+   * Wait for fonts to be loaded.
+   * Ensure that icon fonts are ready to avoid display inconsitencies
+   */
+  await page.waitFor(() => {
```

Comment:
Added this based on https://github.com/garris/BackstopJS/issues/1144#issuecomment-588558285

Needed to ensure that icon fonts have been loaded otherwise icon-only buttons may not display. I actually don't think this is an issue per-se with Backstop and more an issue in general with icon fonts, as they have to load as a separate font request. Icon fonts have [plenty of their own issues](https://cloudfour.com/thinks/seriously-dont-use-icon-fonts/) and longer term I think it'd be worth using inline SVG icons, but for now this is a workaround.